grml-debootstrap

```diff
@@ -1687,30 +1696,20 @@ iface eth0 inet dhcp
 "
 
   if [ -n "$NOINTERFACES" ] ; then
-    einfo "Not installing /etc/network/interfaces as requested via --nointerfaces option" ; eend 0
+    einfo "Not installing /etc/systemd/network/80-dhcp.network as requested via --nointerfaces option"
+    eend 0
   elif [ -n "$USE_DEFAULT_INTERFACES" ] ; then
-    einfo "Installing default /etc/network/interfaces as requested via --defaultinterfaces options."
+    einfo "Installing default /etc/systemd/network/80-dhcp.network as requested via --defaultinterfaces options."
+    echo "$DEFAULT_SYSTEMD_NETWORKD" > "${MNTPOINT}/etc/systemd/network/80-dhcp.network"
     echo "$DEFAULT_INTERFACES" > "${MNTPOINT}/etc/network/interfaces"
```

Comment:
Isn't this wrong? (Now installing the 80-dhcp.network for systemd-network *and* the /e/n/ for ifupdown)

Reply:
Right. The info line could be improved to reflect the current behavior.